jwst/extract_1d/tests/test_expected_skips.py

```diff
@@ -0,0 +1,21 @@
+import os
+import stdatamodels.jwst.datamodels as dm
+from jwst.extract_1d import Extract1dStep
+from jwst.photom import PhotomStep
+
+data_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), 'data'))
+TEST_FILES = {
+    'SOSS/FULL': os.path.join(data_dir, 'jw01209001001_03101_00001_nis_pathloss_dummy.fits'),
```

Comment:
We usually frown on putting FITS files in the repo to use with unit tests. It would be better to use something like the `create_input` function used in the photom unit tests to create a datamodel on the fly that has the appropriate meta data set (i.e. model.meta.subarray.name='FULL') to trigger the expected skip.